packages/ui/src/views/project/create-project-page.tsx

```diff
@@ -157,22 +168,34 @@ export const CreateProjectPage: FC<CreateProjectPageProps> = props => {
             />
           </Fieldset>
 
-          <Button
-            className="mt-3 w-full"
-            borderRadius="full"
-            type="submit"
-            size="md"
-            loading={isLoading}
-            disabled={hasError}
-          >
-            {isLoading
-              ? t('views:createProject.create.projectCreation', 'Creating project...')
-              : t('views:createProject.create.createProject', 'Create project')}
-          </Button>
+          <ControlGroup type="button">
+            <Button
+              className="mt-3 w-full"
+              borderRadius="full"
+              type="submit"
+              size="md"
+              loading={isLoading}
+              disabled={hasError}
+            >
+              {isLoading
+                ? t('views:createProject.create.projectCreation', 'Creating project...')
+                : t('views:createProject.create.createProject', 'Create project')}
+            </Button>
+
+            <div className="mt-3 flex items-center justify-center gap-2">
+              <div className="border-cn-borders-3 w-[145px] shrink border-t" />
+              <span className="text-cn-foreground-3 text-sm">{t('views:createProject.or', 'or')}</span>
+              <div className="border-cn-borders-3 w-[145px] shrink border-t" />
+            </div>
+
+            <Button asChild className="mt-3 w-full" borderRadius="full" type="button" variant="outline" size="md">
+              <Link to="/import">{t('views:createProject.importProject', 'Import project')}</Link>
```

Comment:
this url should not be hardcoded in a view.
it should be coming via a prop like `toProjectImport: () => string`